pages/common/lab.md

```diff
@@ -0,0 +1,19 @@
+# lab
+
+> A CLI for Gitlab.
+
+- Create a merge request and specify remote and target branch:
+
+`lab mr create {{remote}} {{target_branch}}`
+
+- Create a pull request on a github repo:
+
+`lab pull-request`
```

Comment:
Just a note, this and all the below functions are actually https://github.com/github/hub commands. `lab` can call into `hub` commands when its installed.